modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/services/syncml/impl/SyncmlServiceImpl.java

```diff
@@ -346,12 +344,6 @@ private boolean enrollDevice(Document request) throws WindowsDeviceEnrolmentExce
                 }
                 Device generateDevice = generateDevice(DeviceManagementConstants.MobileDeviceTypes.
```

Comment:
generateDevice is not a proper name for a variable. do not include verbs or actions as variable names